instruqt/distributing-with-replicated/01-preparing-to-use-the-sdk/assignment.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: preparing-to-use-the-sdk
+id: kkhcf1j4eaym
+type: challenge
+title: Preparing to Use the SDK
+teaser: Getting ready to use the Replicated SDK
+notes:
+- type: text
+  contents: Let's get ready to use the Replicated SDK
+tabs:
+- title: Shell
+  type: terminal
+  hostname: shell
+difficulty: basic
+timelimit: 300
+---
+
+👋 Introduction
+===============
+
+The Replicated SDK is implemented as a small service that runs
+alongside your application and enables access to the Replicated
+Platform. The SDK allows you to enforce your entitlements and
+take advantage of the telemetry that Replicated provides to help
+you better understand customer instances. It also allows your
+to get information about your application and it's releases, 
+including doing updates checks and showing version history.
+
+
+✅ Preparing to use the SDK
+===========================
+
+To make use of the Replicated SDK, you'll need a couple of
+things. They've been set up for you in this lab environment.
+
+1. A Helm chart for your application. We're going to use the
+   Open Source Bitnami Helm chart for the Harbor container registry
+   for this lab, but it could be the Helm chart for your product
+   or any other Helm chart you wanted to distribute this way.
+2. Access to the Replicated Vendor Portal. You've been given
+   access for the duration of this lab with the username
+   `[[ Instruqt-Var key="USERNAME" hostname="shell" ]]` and
+   the password `[[ Instruqt-Var key="PASSWORD" hostname="shell" ]]`
```

Comment:
I saw this and I was concerned that I would need to save these details or do something with them. What do you think about adding a comment like `Don't worry about saving these, they'll be presented again when you need them`. OR `you should log into the vendor portal with these now at LINK`

Reply:
Probably leaving them out makes more sense, we'd kind of gotten in the habit of putting them in the first lab challenge but they don't really need to be there.